pkg/api/v1alpha1/cluster.go

```diff
@@ -572,11 +573,17 @@ func validateGitOps(clusterConfig *Cluster) error {
 	if gitOpsRef == nil {
 		return nil
 	}
-	if gitOpsRef.Kind != GitOpsConfigKind {
+
+	if gitOpsRef.Kind == FluxConfigKind && !features.IsActive(features.GenericGitProviderSupport()) {
+		return fmt.Errorf("FluxConfig and the generic git provider are not currently supported")
+	}
+
+	if gitOpsRef.Kind != GitOpsConfigKind && gitOpsRef.Kind != FluxConfigKind {
 		return errors.New("only GitOpsConfig Kind is supported at this time")
```

Comment:
should we also mention `FluxConfig` is supported under feature gate?

Reply:
yeah, I was going to wait to add it until we removed the feature gate to avoid potential confusion, but can update it to indicate that it's available under the feature flag

Reply:
https://github.com/aws/eks-anywhere/pull/1827/commits/89624bcff876de204a0cfbb8433650548caa81b0